src/overTime.c

```diff
@@ -70,23 +70,22 @@ void initOverTime(void)
 	// Get current timestamp
 	time_t now = time(NULL);
 
-	// The last timestamp (overTime[149]) should be the last interval of this hour
-	// If the current time is 09:35, the last interval is 09:50 - 10:00 (centered at 09:55)
-	time_t timestamp = now - now % 3600 + 3600 - (OVERTIME_INTERVAL / 2);
+	// Get the centered timestamp of the current timestamp
+	time_t first_slot_ts = now - now % OVERTIME_INTERVAL + (OVERTIME_INTERVAL / 2);
+	time_t last_slot_ts = first_slot_ts + (OVERTIME_SLOTS-1) * OVERTIME_INTERVAL;
```

Comment:
The logic in this PR is broken, unfortunately, I didn't see this before. We'll revert the PR for now. The idea can be re-applied later when we have a solution and proper time for testing.

The issue here is, basically, that `first_slot_ts` should be the one 24 hours in the past, however, it is a timestamp centered at the current timestamp (as the comment correctly says). The `last_slot_ts` is then 24 hours *in the future* which is wrong.

This wasn't an issue in the code before the change as the old code was going *backwards* from now. The new code tries to go forward but start at *now* causing the history to incorrectly love into the future rather than into the past.